CONTRIBUTING.md

```diff
@@ -40,6 +42,10 @@ Issues labeled [help wanted](https://github.com/medic/cht-core/labels/Help%20wan
 Is our documentation up to date? Have we covered everything we should? Could our wording be improved? Read our [Documentation Style Guide](https://docs.communityhealthtoolkit.org/contribute/docs/style-guide/) then open a pull request with your suggested changes or additions.
 Want to talk about Documentation generally? Join our [Community Forum](https://forum.communityhealthtoolkit.org)!
 
+### Translations
+
+If you are a translator but not a developer, we understand that you may need extra help to follow the [process of translating](https://docs.communityhealthtoolkit.org/apps/reference/translations/) software for the first time. If that is the case, please open an issue on the GitHub repo or start a topic on the community forum.
```

Comment:
We may want links here if it's decided that this PR contains desired changes and after the text is decided upon.

Reply:
Good point @andy5995 . @CescaGarrett is the code owner for this page so I have requested her review